.dockerignore

```diff
@@ -0,0 +1,3 @@
+.idea
```

Comment:
Perhaps `.DS_Store` as well?

Reply:
Good point, also added now `integrations-registry`. Seems like I did a bad copy of the .gitignore file.